test/webdriverio/test/actions_test.ts

```diff
@@ -70,4 +84,133 @@ suite('Menus test', function () {
       'The menu should not be openable during a move',
     );
   });
+
+  test('Block menu via keyboard displays expected items', async function () {
+    await tabNavigateToWorkspace(this.browser);
+    await focusOnBlock(this.browser, 'draw_circle_1');
+    await this.browser.keys([Key.Ctrl, Key.Return]);
+    await this.browser.pause(PAUSE_TIME);
+
+    chai.assert.deepEqual(
+      await getContextMenuItemNames(this.browser),
+      [
+        'Duplicate',
+        'Add Comment',
+        'External Inputs',
+        'Collapse Block',
+        'Disable Block',
+        'Delete 2 Blocks',
+        'Move Block',
+        'Edit Block contents',
+        'Cut',
+        'Copy',
+        'Paste',
+      ],
```

Comment:
This test appears to enforce an ordering.  If we are concerned about the order, should we not also be concerned about presence of spacers, shortcut keys, etc.?

Reply:
I rebased my branch and now I'm using the same strategy to verify menu items as other tests, including shortcut keys.